### ISSUE TITLE
exceptions.pyi doesn't contain MultipleInvalid

### ISSUE THREAD

**Issue by wRAR** [Shoobx/mypy-zope#102]
Looks like stubs were generated for an old versin of zope.interface, is that intended?

**Comment by kedder** [comment]
Well, at the time stubs were generated, `zope.interfaces` wasn't old. I'm trying to update them in #105.

**Comment by wRAR** [comment]
Yeah, I guess my question should have read "is it intended that they aren't regenerated". Thanks!

**Comment by kedder** [comment]
Regenerating stubs isn't trivial, because they actually contain the manually added typing information, that is lost when regenerated from original untyped sources. Zope stubs didn't have a lot of these changes, so it is feasible to regenerate and retype again. Hopefully we don't have to do this very often.
